hack/quickstart/openstack/openstack.tf

```diff
@@ -0,0 +1,102 @@
+resource "openstack_compute_secgroup_v2" "kubernetes_control_public" {
+  name = "80AC0F3A-ECF8-424C-88FA-D8571F0C0EC9"
```

Comment:
I believe this is arbitrary and optionally used for back-references, did you auto-generate the names here @philips?